app/models/barcode_printer_type1_d_tube.rb

```diff
@@ -4,6 +4,7 @@
 #
 class BarcodePrinterType1DTube < BarcodePrinterType
   def self.first
+    # Update this?
```

Comment:
Are we using new template names? If so then yep, update them here, otherwise remove this comment.